dags/nocp/scripts/run_ocm_api_load.sh

```diff
@@ -5,7 +5,14 @@ set -x
 run_ocm_api_load(){
     echo "Cleanup previous UUIDs"
     cd ~/
-    export $(cat environment.txt | xargs)
+    # Export environment variables provided by airflow, however don't override local shell variables like PWD
+    env > /tmp/local_env.txt
```

Comment:
Not to nitpick, but something like the below, so you can always dump the values in a new "temp" directory 
and avoid issues..

```
TEMP_DIR=`mktemp -d`
env > /tmp/${TEMP_DIR}/local_env.txt
    while IFS="=" read -r key value; do
        if ! grep -q "$key=" /tmp/${TEMP_DIR}local_env.txt; then
                export $key=$value
        fi
    done < environment.txt
```